plugin/db/spanner/spanner.go

```diff
@@ -49,6 +49,8 @@ type Driver struct {
 	connCtx  db.ConnectionContext
 	client   *spanner.Client
 	dbClient *spannerdb.DatabaseAdminClient
+
+	dbName string
```

Comment:
How about using `currentDatabase`

Reply:
That's too long IMO. I'll add a comment instead.

Reply:
```suggestion
	// dbName is the currently connected database name.
	dbName string
```